### DISCUSSION TITLE
Problem with display 3d Canvas in mobile

### DISCUSSION THREAD

**Discussion by abdallaamrfawzy** [community/community#135463] [Programming Help]
### Body

I developed my website using Recat .js, @react-three/fiber
And the 3d drawing is displayed on my laptop
But when I open the website on the mobile, a white screen appears. I don't know what the problem is
Knowing that I tried through my laptop to convert the screen size to a mobile screen through inspect and it worked
I want to know what is the reason behind this problem
my website is https://abdalla-amr.web.app/

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by everforyou** [comment]
Please make sure you have the correct viewport meta tag in your HTML.
```html
<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

**Reply by abdallaamrfawzy** [reply]
i have already added it
